apps/examples/mpu/tc_write_data.c

```diff
@@ -15,8 +15,8 @@
  * language governing permissions and limitations under the License.
  *
  ****************************************************************************/
-/// @file tc_write.c
-/// @brief Test Case Example for write to kernel space from user space
+/// @file tc_write_data.c
+/// @brief Test Case Example for write to kernel data space from user space
```

Comment:
Lets not use C++ style of comment in our files. Lets maintain consistency across file. applies to all files